components/Avatars/Avatar.css

```diff
@@ -0,0 +1,97 @@
+
+ .Avatar_container{
+   width: 90%;
+   height: 10rem;
+   display: flex;
+   justify-content:center;
+   align-items: center;
+   border: 1px solid grey;
+   margin: 2rem 0;
+ }
+ 
+ 
+ .avatar-xl{
+   width:7em;
+   height:7em;
+   border-radius:50%;
+  
+ }
+ .avatar-lg{
+   width:5em;
+   height:5em;
+   border-radius:50%;
+  }
```

Comment:
You can use CSS variable for `border-radius` property, since it is getting reused multiple times.